src/js/app/App.js

```diff
@@ -0,0 +1,48 @@
+import routes from 'js/utils/routes';
+import { Header } from 'js/components';
+import { Home, Lines, Login, Sections, NotFound, Stations } from 'js/pages';
+import render from 'js/utils/render';
+import ajax from 'js/api';
+
+const { HOME, LINES, LOGIN, SECTIONS, STATIONS } = routes;
+
+const App = (async () => {
+  let stations = await ajax.getStations();
+  const setStations = newStations => {
+    stations = newStations;
+    render();
+  };
```

Comment:
서버에서 받아오신 station 정보를 station 변수에 저장하고 계신데용~ state 객체에 저장하면 어떨까요?? 서버에서 받아온 상태값을 여기저기 변수에 흩뿌려놓기보다는, state 객체에 모아두면 관리,변경하기 쉽지 않을까요? ㅎㅎ 
 
```suggestion
  let stations = await ajax.getStations();
  let state = {
      stations 
  } 
  const setState= newState=> {
    state = newState
    render();
  };
```

Reply:
사실 이거에 관해서도 이번에 첫 상태가 들어갈때 고민을 했었는데... 많은 의견이 있는것 같습니다. 이렇게 변수에 흩뿌려놓으면 말씀하신대로 더 많은 props를 전달하거나 그 상태를 변경해주는 함수를 더 많이 생성해야할수도 있습니다. 하지만 상태를 변경하는 코드는 단순해지고 즉, 복잡성이 낮습니다.
하지만 객체로 관리하게되면 그것이 하나의 묶음으로 다니기때문에 참조나 변경을 할때 용이하지만 변경을 하는 코드는 객체의 depth가 깊어졌을때 복잡도가 올라간다고 생각합니다!

아 위의 과정은 물론 불변성을 유지한다고 했을때의 가정입니다ㅎㅎ 문희님 말씀하신대로 해보면서 immer도 같이 한번 써봐야겠어요!